packages/fxa-auth-client/lib/client.ts

```diff
@@ -1249,6 +1249,19 @@ export default class AuthClient {
     );
   }
 
+  async getRecoveryKeyHint(uid: hexstring) {
+    return this.request('GET', `/recoveryKey/hint?uid=${uid}`);
```

Comment:
I'd used GET because we're looking to retrieve information and not posting to the DB, but I assume the preference for POST is to avoid passing in params? @vbudhram Does it make a difference if the param we're passing is `email` or `uid`?

We don't have any tickets yet for enabling hint retrieval on the client, so I'll add a TODO to modify this method when we get to that point.

Reply:
I've filed [FXA-7400](https://mozilla-hub.atlassian.net/browse/FXA-7400) to switch from GET to POST before we need to retrieve the key on the front end.